robot_smach_states/src/robot_smach_states/navigation/navigate_to_grasp.py

```diff
@@ -6,28 +6,29 @@
 import rospy
 
 # TU/e Robotics
-from cb_planner_msgs_srvs.srv import *
 from cb_planner_msgs_srvs.msg import *
-from robot_skills.arms import Arm
+from robot_skills.arms import PublicArm
```

Comment:
Personally, I think it would have been much nicer if the arm you import was called `Arm` and the other one `PrivateArm` (so the other way around).